docs/developers/contributing/testing.md

```diff
@@ -205,38 +205,56 @@ you create during testing need to be cleaned up at the end of each test. We thus
 recommend that you use the following fixtures when needing a widget or
 {class}`~napari.Viewer` in a test.
 
-#### qtbot
+#### qapp and qtbot
 
-If you need a `QApplication` to be running for your test, you can use the
+If you need a `QApplication` to be running for your test you can use the
+[`qapp`](https://pytest-qt.readthedocs.io/en/latest/reference.html#module-pytestqt.plugin)
+fixture from [`pytest-qt`](https://pytest-qt.readthedocs.io/en/latest/index.html),
+a napari testing dependency.
+
+If you need to instantiate a Qt object (e.g., a widget) for your test, we recommend
+that you use the
 [`qtbot`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot)
-fixture from `pytest-qt`, a napari testing dependency.
+fixture. `qtbot`, which itself depends on `qapp` , allows you to test user input
+(e.g., mouse clicks) by sending events to Qt objects.
 
 ````{note}
 Fixtures in pytest can be a little mysterious, since it's not always
-clear where they are coming from.  In this case using the `pytest-qt` `qtbot`fixture
-looks like this:
+clear where they are coming from.  The `pytest-qt` `qapp` and `qtbot` fixtures
+can be used in two ways; by adding them to the list of arguments of your test function:
 
 ```python
-# just by putting `qtbot` in the list of arguments
-# pytest-qt will start up an event loop for you
 def test_something(qtbot):
     ...
 ```
+or by using pytest's `usefixtures`, which avoids adding an unused argument to your
+test function:
+
+```python
+@pytest.mark.usefixtures('qtbot')
+def test_something():
+    ...
+```
 <br/>
 ````
 
-`qtbot` provides a convenient
+`qtbot` also provides a convenient
 [`addWidget`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot.addWidget)
```

Comment:
```suggestion
[`add_widget`/`addWidget`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot.addWidget)
```

Reply:
Do you want both or is there one that you prefer? I guess `add_widget` would be more 'python-ic'?

Reply:
Looks okay as both so will leave

Reply:
In our test we are using both of them. So I would like to signal that both are valid.